elasticsearch/src/test/java/org/apache/calcite/adapter/elasticsearch/AggregationTest.java

```diff
@@ -47,7 +46,6 @@
 /**
  * Testing Elasticsearch aggregation transformations.
  */
-@Disabled("RestClient often timeout in PR CI")
```

Comment:
so this comment is false statement ?

Reply:
Yes from my test. After adding the translation of 'SEARCH' RexCall, all the tests can pass.